roles/openshift_certificate_expiry/README.md

```diff
@@ -104,35 +139,71 @@ $ ansible-playbook -v -i HOSTS /usr/share/ansible/openshift-ansible/playbooks/ce
 
 > [View This Playbook](../../playbooks/certificate_expiry/easy-mode.yaml)
 
-***
+### Easy mode and upload reports to masters
+
+This example builds on top of [easy-mode.yaml](#easy-mode) and additionally
+uploads a copy of the generated reports to the masters, with a timestamp in the
+file names.
+
+This is specially useful when the playbook runs from within a container, because
+the reports are generated inside the container and we need a way to access them.
+Uploading a copy of the reports to the masters is one way to make it easy to
+access the reports. Alternatively you can use the
+[role variables](#role-variables) that control the path of the generated reports
+to point to a container volume.
 
-Default behavior:
+With the container use case in mind, this playbook allows control over some
+options via environment variables:
+
+ - `CERT_EXPIRY_WARN_DAYS`: sets `openshift_certificate_expiry_warning_days`,
```

Comment:
You make an excellent point about the default warning window there. I did not think of that condition you described.

:+1: to keeping the new values at 45.

> On one hand I wanted to show how to change the default - I could do this by setting the value to
> the same as the default, but I think it looks more clear if the value differs.

:+1: 

> Even in less extreme cases, I believe cluster admins prefer to have a fairly advanced warning in order
> to have time to plan cert redeployment. In fact I was wondering if the role's default should be increased,
> with this example's value set even higher :) What do you think?

Totally. I'm opening another PR so we can track this for later.

https://github.com/openshift/openshift-ansible/issues/3856